REvernus/ViewModels/MarginToolViewModel.cs

```diff
@@ -281,16 +281,13 @@ private void WatcherOnChanged(object sender, FileSystemEventArgs e)
                 {
                     using (var reader = new StreamReader(file))
                     {
-                        int lineCounter = 0;
-                        while (!reader.EndOfStream)
+                        try
                         {
-                            try
+                            var firstline = reader.ReadLine().Split(','); // read first line and disregard
```

Comment:
No need to assign a variable to the output, the firstline variable is not used, and delimiting is also not necessary. simply `reader.ReadLine()` will work fine here.